### ISSUE TITLE
Make it clearer when TF is waiting for a VM to get an IP

### ISSUE THREAD

**Issue by jantari** [nutanix/terraform-provider-nutanix#235]
**Is your feature request related to a problem? Please describe.**
I've recently encountered the case where I was creating a VM in a subnet without DHCP, and terraform got stuck on:

    Still creating ... [ x time elapsed ]

forever. I tried many times including with debug level `TF_LOG=TRACE` but all it was that the provider was "waiting on vm".
Meanwhile, all tasks associated with the VMs creation were successful and the VM was powered up and running.

In the end, I found out it's because the TF provider waits for the VM to get an IP, which never happened.

**Describe the solution you'd like**
I think "Still creating" or "waiting for VM" is misleading when the VM is fully operational and all tasks are finished.
I see no issues with waiting for an IP - although it's not strictly needed and could be made optional - but the output of the provider should make the distinction clear. Something like:

    VM created, waiting for IP ... [ x time elapsed ]

**Describe alternatives you've considered**
Make the fact it waits for an IP better documented and introduce an ability to skip it, but still it should be in the DEBUG or TRACE output.

**Additional context**
VM that's not getting an IP:
![vm_without_ip](https://user-images.githubusercontent.com/25607767/105152786-efbd6e80-5b07-11eb-8ac7-36b309775206.png)

Terraform:
![terraform](https://user-images.githubusercontent.com/25607767/105152851-006de480-5b08-11eb-8d3e-e119e91aa897.png)

EDIT:

Software | Version
----------|----------
Prism Central | pc.2020.11.0.1
terraform | v0.14.4
nutanix provider | v1.1.1

**Comment by TheMoves** [comment]
Were you able to pass an IP to the VM? i've tried using Cloud Init and as far as I can tell it doesn't get passed to the VM.

**Comment by jantari** [comment]
@TheMoves yes, it's unrelated to this issue but [this helped me get it working](https://www.burgundywall.com/post/using-cloud-init-to-set-static-ips-in-ubuntu-20-04). You could also just use `write_files:` in cloud-init to write a file with the config you want to `/etc/netplan/`
